x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/sample_response/sample_response.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiCodeBlock,
+  EuiFieldSearch,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ResultSettingsLogic } from '../result_settings_logic';
+
+import { SampleResponseLogic } from './sample_response_logic';
+
+export const SampleResponse: React.FC = () => {
+  const { reducedServerResultFields } = useValues(ResultSettingsLogic);
+
+  const { query, response } = useValues(SampleResponseLogic);
+  const { queryChanged, getSearchResults } = useActions(SampleResponseLogic);
+
+  useEffect(() => {
+    getSearchResults(query, reducedServerResultFields);
+  }, [query, reducedServerResultFields]);
+
+  return (
+    <EuiPanel hasShadow>
```

Comment:
[change request on Davey's behalf 😄]

```suggestion
    <EuiPanel hasBorder>
```